### PR TITLE
[153] Style the Product Details Page using CSS-Fullstack

### PR DESCRIPTION
**Pull request by Constancia2**
Desktopview 1
desktopview 2
mobile view 1
mobileview 2
mobileview 3

### CODE REVIEW

**Comment by Ireti203** [comment]
Kindly close this PR and create a new one using the title on the app

File: submissions/OnyediriConstancia/HTML/index.html [hunk 1]
```diff
@@ -4,40 +4,152 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Shopfood</title>
+    <link rel="stylesheet" href="/product.css">
 </head>
 <body>
-  <h2>SHOPFOOD</h2>  
-  <a href="#">view other foodstuff here</a> <br>
+  <header>
+
```
Comment (Ireti203) [inline submissions/OnyediriConstancia/HTML/index.html:11]: Remove whitespace

File: submissions/OnyediriConstancia/HTML/index.html [hunk 2]
```diff
@@ -4,40 +4,152 @@
...
     <title>Shopfood</title>
+    <link rel="stylesheet" href="/product.css">
 </head>
 <body>
-  <h2>SHOPFOOD</h2>  
-  <a href="#">view other foodstuff here</a> <br>
+  <header>
+   
+    <div  class="navbar">
+
```
Comment (Ireti203) [inline submissions/OnyediriConstancia/HTML/index.html:13]: Remove whitespace

File: submissions/OnyediriConstancia/HTML/index.html [hunk 3]
```diff
@@ -4,40 +4,152 @@
...
  
-  <img src="/Images/tomatoes.jpg" alt="Tomatoes" width="190px" height="100px">
-   <h4>Tomatoes, 1kg</h4>
- 
-  <h4><strong>price:</strong>2000 NGN</h4>
+     <ul>
+       <a href="aboutUs.html"> <li>About Us</li></a>
+       <a href="ContactForm.html">  <li>Contact Form</li></a>
+       <a href="product.html"> <li>Product</li></a> 
+
```
Comment (Ireti203) [inline submissions/OnyediriConstancia/HTML/index.html:23]: Remove whitespace

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Comment by adaugochi** [comment]
@Constancia2 on the **Dufuna app**, is this ` [153] Style the Product Details Page using CSS-Fullstack` what is showing as your task title?

**Pull request closed without merging**